homeassistant/components/neato/.translations/de.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
These files will be overridden by the translations download script. Lokalise.com is where they should be entered.

Reply:
Does the entire folder get overriden or just non english languages?

Reply:
@balloob should I delete those files?

Reply:
Yes. The English default translation can be kept.

Reply:
done